### PR TITLE
Issue #203 pneumatics wiring 

### PR DESCRIPTION
**Pull request by samneff72**
Proposed fix for issue #203: add a reference to "use the rules" & a link to the Pneumatics Manual.
None of the hardware technical information should be included in this article since the focus is wiring the pneumatics. Clarified "Additional PCMs" sections as well. 

Internal link is formatted correctly. 

TODO: More images? 
11/2 edit: Bailing on adding more images. Someone else can take this on. 

These changes were originally placed in PR #218

### CODE REVIEW

**Comment by samneff72** [comment]
@yellowjaguar5 I've gotten as far as I can, I think I have the right file path and formatting based on the styleguide page and swear the first link is exactly what was passing build checks on my previous PR. I'm stumped and don't know what the "bash code 2" means.

**Comment by yellowjaguar5** [comment]
Huh. I'm confused too. Error 2 means that a link is broken. If you search the page for `error`, you can find that the errors are on [lines 843 and 844](https://dev.azure.com/wpilib/Documentation/_build/results?buildId=6348&view=logs&j=fc8ae5f9-abcc-54af-cfd7-712b49b9dd20&t=d6592ad4-f9a4-5bc1-dd94-e2417343b88e&l=843). @Daltz333, any ideas?

**Comment by yellowjaguar5** [comment]
I noticed that you were wondering if you should add more images to the page. How about one of the overview? If you want to use this, please provide credit. I said ``Image courtesy of `@stefenacepcion <https://www.chiefdelphi.com/t/control-system-layout-infographic/361110/20>`__ on Chief Delphi. `Click here <https://drive.google.com/file/d/1C8mV5wYJSe1oBIUHVwHYJ3nsNioG2019/view>`__ for a high-resolution 30x24 poster version.`` in other parts of the documentation
pneumatics-subsystem-01

**Comment by sciencewhiz** [comment]
> If you want to use this, please provide credit. I said `` Image courtesy of `@stefenacepcion <https://www.chiefdelphi.com/t/control-system-layout-infographic/361110/20>`__ on Chief Delphi. `Click here <https://drive.google.com/file/d/1C8mV5wYJSe1oBIUHVwHYJ3nsNioG2019/view>`__ for a high-resolution 30x24 poster version. `` in other parts of the documentation

I'm not sure that just providing image credit is enough, I think explicit permission from the copyright holder is required.

**Comment by yellowjaguar5** [comment]
Sounds good. Would you like to contact the author or would you like me to do so?

**Comment by samneff72** [comment]
Don't worry about it, I couldn't get the image to work. 
Bailing on my "additional images" todo, request merge review on check completions @AustinShalit

File: source/docs/hardware/hardware-basics/wiring-pneumatics.rst [hunk 1]
```diff
@@ -1,72 +1,43 @@
 Wiring Pneumatics
 =================
 
-.. hint:: Wiring pneumatics has been made very simple in the 2019 Control System. A single Pneumatics Control Module is all that will be needed for many pneumatics applications, with additional PCMs supporting more complex designs including more than 8 solenoid channels or a mix of 12V and 24V solenoids.
+.. hint:: For pneumatics safety & mechanical requirements, consult this year's Robot Construction rules (Section 10 in 2019). For mechanical design guidelines, the FIRST Pneumatics Manual is located `here (last updated 2017) <https://firstfrc.blob.core.windows.net/frc2017/pneumatics-manual.pdf>`__
```
Comment (AustinShalit) [inline source/docs/hardware/hardware-basics/wiring-pneumatics.rst]:
```suggestion
.. hint:: For pneumatics safety & mechanical requirements, consult this year's Robot Construction rules. For mechanical design guidelines, the FIRST Pneumatics Manual is located `here (last updated 2017) <https://firstfrc.blob.core.windows.net/frc2017/pneumatics-manual.pdf>`__
```

**Pull request merged**